docs/security/user-certs.md

```diff
@@ -91,7 +111,8 @@ To solve the problem split the bundle in the two PEM files as documented in the
 
 ### Revoking your user certificate
 
-If the security of your certificate or private key has been compromised, you have a responsibility to revoke the certificate. You can follow the steps [here](../security/certificate-management#osg-user-cert-revoke) to revoke your certificate. The same instructions apply if you need to revoke a certificate because your email address changed or your name has changed.
+If the security of your certificate or private key has been compromised, you have a responsibility to revoke the certificate.
+Contact [ca@cilogon.org](mailto:ca@cilogon.org) in order revoke your certificate. 
```

Comment:
We need to have both revocation instructions and clearly state for which CA they're for

Reply:
Updated